onnx/backend/test/case/model/sequence.py

```diff
@@ -13,6 +13,64 @@
 from typing import List, Optional, Text, Union
 
 
+def SequenceEmptyImpl():  # type: () -> List[Optional[np.ndarray]]
+    return []
+
+
+def SequenceConstructImpl(*tensors):  # type: (*np.ndarray) -> List[np.ndarray]
+    return list(tensors)
+
+
+def SequenceInsertImpl(sequence, tensor, position=None):
+    # type: (List[np.ndarray], np.ndarray, Optional[int]) -> List[np.ndarray]
+    if position is None:
+        position = len(sequence)
+    sequence.insert(position, tensor)
+    return sequence
+
+
+def SequenceAtImpl(sequence, position):
+    # type: (List[np.ndarray], int) -> np.ndarray
+    return sequence[position]
+
+
+def SequenceEraseImpl(sequence, position=None):
+    # type: (List[np.ndarray], Optional[int]) -> List[Optional[np.ndarray]]
+    if position is None:
+        position = -1
+    del sequence[position]
+    return sequence
+
+
+def SequenceLengthImpl(sequence):
+    # type: (List[np.ndarray]) -> np.int64
+    return np.int64(len(sequence))
+
+
+def SplitToSequenceImpl(tensor, split=None, axis=0, keepdims=1):
+    # type: (np.ndarray, Optional[Union[int, List[int]]], int, int) -> List[np.ndarray]
```

Comment:
```suggestion
    # type: (np.ndarray, Optional[Union[int, List[int]]], int, int) -> List[np.ndarray]
```
Can we use numpy type such as `np.int64` as the `int` here?

Reply:
Everywhere else also uses int. I'd prefer using int here as well. In addition as below, it seems some numpy methods don't work well with mypy.